.fixtures.yml

```diff
@@ -20,6 +20,8 @@ fixtures:
     pe_r10k: "git@github.com:puppetlabs/puppetlabs-pe_r10k.git"
     pe_staging: "git@github.com:puppetlabs/puppetlabs-pe_staging.git"
     puppet_enterprise: "git@github.com:puppetlabs/puppetlabs-puppet_enterprise.git"
+    stunnel: "git@github.com:puppetlabs/puppetlabs-stunnel.git"
```

Comment:
There have been minor fixes to the repo and no new release to the forge, so I see no reason not to use the git repo vs forge